phabricatoremails/render/events/common.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from datetime import timezone, timedelta
+from enum import Enum
+from math import floor, ceil
+from typing import Optional, Dict, List
+
+"""Describes data structures that are used by both secure and public revision events.
+
+Additional information about this module can be found in __init__.py
```

Comment:
The data structures in the `render.events` package are distinct enough to be defined in different files, but similar enough to have some common documentation. I don't think that duplicating the documentation is the right call, since that increases the chances of docs going out-of-date.